src/database/migrations/1731879052218-migration-name-placeholder.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class MigrationNamePlaceholder1731879052218
+  implements MigrationInterface
+{
+  name = 'MigrationNamePlaceholder1731879052218';
```

Comment:
change this name to appropriate name that is appropriate for the migration, change also the name of the migration file and class. look at how Begzod and I did in the previous migration files.